README.md

```diff
@@ -1 +1,71 @@
-# TicTacToe
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Tic Tac Toe Setup
+
+> The purpose of this project is to setup the Tic Tac Toe game, for two players, using Ruby.
+
+- This is the Milestone #1, to create a executable file (main.rb).
+
+## Built With
+
+- Ruby
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
**[OPTIONAL]** remove the unnecessary titles, and also work on the getting started section